src/alchemlyb/workflows/abfe.py

```diff
@@ -0,0 +1,665 @@
+import os
+from os.path import join
+from glob import glob
+import pandas as pd
+import numpy as np
+import logging
+import matplotlib.pyplot as plt
+
+from .base import WorkflowBase
+from ..parsing import gmx, amber
+from ..preprocessing.subsampling import decorrelate_dhdl, decorrelate_u_nk
+from ..estimators import BAR, TI
+from ..estimators import AutoMBAR as MBAR
+from ..visualisation import (plot_mbar_overlap_matrix, plot_ti_dhdl,
+                             plot_dF_state, plot_convergence)
+from ..postprocessors.units import get_unit_converter
+from ..convergence import forward_backward_convergence
+from .. import concat
+from .. import __version__
+
+
+class ABFE(WorkflowBase):
+    '''Alchemical Analysis style automatic workflow.
```

Comment:
```suggestion
    '''Workflow for absolute and relative binding free energy calculations.
    
    This workflow provides functionality similar to the ``alchemical-analysis.py`` script.
    It loads multiple input files from alchemical free energy calculations and computes the
    free energies between different alchemical windows using different estimators. It 
    produces plots to aid in the assessment of convergence.
    
```

You can add more and you can rewrite, of course, but add a concise summary. Possibly add links eg to alchemical-analysis.